hal/src/nRF52840/ble_hal_impl.h

```diff
@@ -68,9 +68,12 @@
 #define BLE_MAX_TX_POWER                            (8)
 
 /* Default scanning parameters */
+#define BLE_DEFAULT_SCANNING_TIMEOUT_MS             5000
 #define BLE_DEFAULT_SCANNING_INTERVAL               BLE_MSEC_TO_UNITS(100, BLE_UNIT_0_625_MS)   /* The scan interval: 100ms (in units of 0.625 ms). */
 #define BLE_DEFAULT_SCANNING_WINDOW                 BLE_MSEC_TO_UNITS(50, BLE_UNIT_0_625_MS)    /* The scan window: 50ms (in units of 0.625 ms). */
-#define BLE_DEFAULT_SCANNING_TIMEOUT                BLE_MSEC_TO_UNITS(5000, BLE_UNIT_10_MS)     /* The timeout: 5000ms (in units of 10 ms. 0 for scanning forever). */
+#define BLE_DEFAULT_SCANNING_TIMEOUT                BLE_MSEC_TO_UNITS(BLE_DEFAULT_SCANNING_TIMEOUT_MS, BLE_UNIT_10_MS)     /* The timeout: 5000ms (in units of 10 ms. 0 for scanning forever). */
+// Extended timeout for the scanning timeout guard timer
+#define BLE_SCANNING_TIMEOUT_EXT                    1000
```

Comment:
Let's clarify that this is milliseconds as we are dealing with various time units here.

Reply:
Changed to `BLE_SCANNING_TIMEOUT_EXT_MS`.